### PR TITLE
feature: localize draw

### LINKED ISSUE #2065: Drawing disappears when show length/area or sections
When choosing to show length/area or sections in a drawing the drawing disappears from the map.
image
image

### PR DESCRIPTION
**Pull request by Grammostola**
Aims to fix #2065 (the long way around)

Some testing would be good, around measure and draw. I've done some testing and I have gotten most of the strings and numbers but do tell if there's something missing (or a crash, I've changed a number of function signatures but who likes `formatArea(undefined, 0, undefined, undefined, cat)`?)

 The editor appears to have parallel files and I haven't tested it.

### CODE REVIEW

**Comment by johnnyblasta** [comment]
There is a little bit faulty on the translation.
On the types of points.
Check mark refers to a [example check icons](https://www.flaticon.com/free-icons/check-mark)
Cross mark (also known as 'X mark') refers to a [example cross icons](https://www.flaticon.com/free-icons/cross-mark)
Plus mark refers to a [example plus icons](https://www.flaticon.com/free-icons/plus-mark)
Also notice the blank space separation.

**Comment by Grammostola** [comment]
Thanks for the feedback. I did experience difficulties with the English terms for "kors" and "kryss"-symbols. (Obviously "Ask an LLM" wasn't immediately the right move. Humans are still better at some things :P )

So 
```json
           "pointTypeCross": "Kors",
            "pointTypeCheckmark": "Kryss",
```

can become

```json
           "pointTypePlus": "Kors",
            "pointTypeX": "Kryss",
```

In English

```json
           "pointTypePlus": "Plus",
            "pointTypeX": "Cross",

```
What was that about a blank space separation?

**Comment by johnnyblasta** [comment]
> 
> What was that about a blank space separation?

You had "Checkmark", but the correct would have been "Check mark".
If you don't use "mark" or "sign" as in your suggestion above there is no need for "särskrivning"

**Comment by Grammostola** [comment]
Should be ok now.

**Comment by johnnyblasta** [comment]
> Should be ok now.

Nah, seems to still be confusing between cross and plus.

**Comment by Grammostola** [comment]
Please specify what you would like to see then, I did provide suggestions earlier : )

File: src/loc/en_US.json [hunk 1]
```diff
@@ -80,7 +116,35 @@
...
+            "rotation": "Rotation",
+            "degreesRotation": "Degrees",
+            "marginForText": "Margin",
+            "frame": "Frame",
+            "margin": "Margin",
+            "point": "Point",
+            "pointTypePoint": "Point",
+            "pointTypeCircle": "Circle",
+            "pointTypeX": "X",
+            "pointTypePlus": "Cross",
```
Comment (johnnyblasta) [inline src/loc/en_US.json:143]: "pointTypePlus": "Cross" should be "pointTypePlus": "Plus"

**Review by johnnyblasta — COMMENTED** [review]
Here is what I meant was wrong.

File: src/loc/en_US.json (hunk 1 above)
Comment (johnnyblasta) [reply src/loc/en_US.json:143]: "Kors" works in swedish and "Plus" would also be appropriate in swedish, so I don't know which is best.

**Pull request merged**